cluster-scaling/README.adoc

```diff
@@ -0,0 +1,89 @@
+= Kubernetes Cluster Scaling
+:toc:
+:imagesdir: ../images
+
+This tutorial will walk you through how to setup cluster scaling using `cluster-autoscaler` to enable worker node autoscaling based on multiple metrics within Kubernetes.
+
+== Preliminary Steps
+
+Each worker running in your Kubernetes cluster MUST have the necessary IAM Roles assigned. These differ base on how you'd like to configure the autoscaler.
```

Comment:
Can this be linked from the cluster creation instructions as well? Also, if the cluster has already been created, how would the user proceed?

Reply:
you can modify existing IAM permissions to an ec2 instance and add new IAM roles to existing instances.

Reply:
Great feedback, these instructions are assuming you didn't use the kops addon for CA. I will clarify and make sure that people know they can configure on bring up as well.